static/css/colors.css

```diff
@@ -9,17 +11,25 @@
 }
 
 .custom-click-row {
+    --h: 88px;   /* the size of the image */
+    --b: 20px;     /* the border thickness*/
+    --g: 0px;    /* the gap */
+    --c: #4ECDC4; /* the color */
     background-color: var(--high-contrast);
-    a:link {
-        color: var(--nav-bar);
-    }
-    a:visited {
-        color: var(--nav-bar);
-    }
-    a:hover {
-        color: var(--darker-neutral);
-    }
+    outline: calc(var(--h)/2) solid #0009;
+    outline-offset: calc(var(--h)/-2);
+    transition: .4s;
+    color: var(--high-contrast);
+    border-radius:10px;
+
+}
+.custom-click-row:hover {
```

Comment:
I won't block this from going through because of this, but found it interesting... I don't think hover is going to be a good/functional experience on mobile.

Mobile web dev is not something I have ever really done but one of the hardest parts about UI/UX is the wide variety of methods that can be used to view the site.
![433996326_1171350744030990_8465594348915043779_n](https://github.com/jmirving/cpm-detail/assets/25354177/a14b63ba-3775-42b9-9c85-ac32431c6b19)

Reply:
What. We doing cross platform. Please send my Mac testing device. @jmirving